prow/github/client.go

```diff
@@ -1002,3 +1002,27 @@ func (c *Client) Merge(org, repo string, pr int, details MergeDetails) error {
 
 	return nil
 }
+
+// ListCollaborators gets a list of all users who have access to a repo (and can become assignees
+// or requested reviewers). This includes, org members with access, outside collaborators, and org
+// owners.
+func (c *Client) ListCollaborators(org, repo string) ([]User, error) {
```

Comment:
How many are there for k/k?

Reply:
535. A bunch of the users don't appear to be real people though, see this [query](https://github.com/orgs/kubernetes/people?utf8=%E2%9C%93&query=k8s-).

Reply:
So I guess each `ListCollaborators` call takes 6 tokens. Need to make sure that's not a problem. If it is then we can add some caching behind an `IsCollaborator` call.

Reply:
I like the idea of adding an http cache as a separate Prow component once we start running into rate limits and routing all github api calls through it.  That would cache the `ListCollaborators` response (and other responses) between **all** instances of hook too which is substantial since we have 4 instances of hook deployed.